docs/dag-run.rst

```diff
@@ -0,0 +1,197 @@
+    You can use an online editor for CRON expressions such as `Crontab guru <https://crontab.guru/>`_
+
+Alternatively, you can also use one of these cron "presets":
+
++--------------+----------------------------------------------------------------+---------------+
+| preset       | meaning                                                        | cron          |
++==============+================================================================+===============+
+| ``None``     | Don't schedule, use for exclusively "externally triggered"     |               |
+|              | DAGs                                                           |               |
++--------------+----------------------------------------------------------------+---------------+
+| ``@once``    | Schedule once and only once                                    |               |
++--------------+----------------------------------------------------------------+---------------+
+| ``@hourly``  | Run once an hour at the beginning of the hour                  | ``0 * * * *`` |
++--------------+----------------------------------------------------------------+---------------+
+| ``@daily``   | Run once a day at midnight                                     | ``0 0 * * *`` |
++--------------+----------------------------------------------------------------+---------------+
+| ``@weekly``  | Run once a week at midnight on Sunday morning                  | ``0 0 * * 0`` |
++--------------+----------------------------------------------------------------+---------------+
+| ``@monthly`` | Run once a month at midnight of the first day of the month     | ``0 0 1 * *`` |
++--------------+----------------------------------------------------------------+---------------+
+| ``@yearly``  | Run once a year at midnight of January 1                       | ``0 0 1 1 *`` |
++--------------+----------------------------------------------------------------+---------------+
+
+Your DAG will be instantiated for each schedule along with a corresponding 
+DAG Run entry in backend.
+
+.. note::
+
+    If you run a DAG on a schedule_interval of one day, the run stamped 2020-01-01 
+    will be triggered soon after 2020-01-01T23:59. In other words, the job instance is 
+    started once the period it covers has ended.  The execution_date passed in the dag 
+    will also be 2020-01-01.
+
+    The first DAG Run is created based on the minimum ``start_date`` for the tasks in your DAG. 
+    Subsequent DAG Runs are created by the scheduler process, based on your DAG’s ``schedule_interval``, 
+    sequentially. If your start_date is 2020-01-01 and schedule_interval is @daily the first run 
+    will be created on 2020-01-02 i.e. after your start date has passed.
+
+Re-run DAG
+''''''''''
+There can be cases where you will want to execute your DAG again. One such case is when the scheduled
+DAG run fails. Another can be the scheduled DAG run wasn't executed due to low resources or the DAG being turned off.
```

Comment:
Low resources shouldn't stop a dag run from being created, so lets remove that bit.